templates/form/index.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello FormController!{% endblock %}
+
+{% block body %}
+
+    <div class="bg-light m-4">
+        <table class="table">
+            <tbody>
+            <tr>
+                <th scope="row">Projektas</th>
+                <th scope="row">Studentas</th>
+            </tr>
+            <tr>
+                <td>{{ project|escape }}</td>
+                <td>{{ name }}</td>
+            </tr>
+            </tbody>
+        </table>
+        {% if name == 'Miroslav' %}
+            <div class="alert alert-success" role="alert">
+                Dešimt balų
+            </div>
+        {% else %}
+            <div class="alert alert-warning" role="alert">
+                Gal pasiseks kitą kartą
+            </div>
+        {% endif %}
+    </div>
+    <div class="m-4">
+        <a class="text-light" href="/">Visi studentai</a>
```

Comment:
Pagrindinio puslapio adresas gali pasikeisti.
Todėl yra geresnė praktika naudoti `<a href="{{path(home)}}">` vietoj jau apskaičiuotos reikšmės `<a href="/">`

Plačiau:
 * https://symfony.com/doc/current/templating.html#linking-to-pages
 * https://symfony.com/doc/current/routing.html